src/azure-cli/azure/cli/command_modules/network/_template_builder.py

```diff
@@ -29,6 +29,8 @@ def _build_frontend_ip_config(cmd, name, public_ip_id=None, subnet_id=None, priv
 
     if zone and cmd.supported_api_version(min_api='2017-06-01'):
         frontend_ip_config['zones'] = zone
+    elif not zone and cmd.supported_api_version(min_api='2020-08-01'):
+        frontend_ip_config['zones'] = []
```

Comment:
`frontend_ip_config['zones']` will be assigned twice if API version is 2020-08-01